### PR TITLE
ADDITION: uTorrent

### PR DESCRIPTION
**Pull request by bcurran3**
**What this PR does / why we need it**:

Adds uTorrent, providing an alternative to Transmission.

**Which issue (if any) this PR fixes**:

Fixes # n/a

**Any other useful info**:

My preference over Transmission.

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -368,6 +369,17 @@ openvpn_password: secretpassword
...
 
+###
+### uTorrent
+###
+utorrent_available_externally: "false"
+utorrent_config_directory: "{{ docker_home }}/utorrent/config"
+utorrent_download_directory: "{{ downloads_root }}"
+utorrent_port_http: "8111"
+utorrent_port_bt: "6881"
+utorrent_user_id: "999"
```
Comment (davestephens) [inline group_vars/all.yml:380]: What uid is this?

File: group_vars/all.yml [hunk 2]
```diff
@@ -368,6 +369,17 @@ openvpn_password: secretpassword
...
+utorrent_group_id: "1001"
```
Comment (davestephens) [inline group_vars/all.yml:381]: And gid?

Comment (bcurran3) [reply group_vars/all.yml:381]: Typo, smartypants!

File: group_vars/all.yml (hunk 1 above)
Comment (bcurran3) [reply group_vars/all.yml:380]: Jumping on it!

File: group_vars/all.yml (hunk 2 above)
Comment (bcurran3) [reply group_vars/all.yml:381]: I think you're looking at an old commit. I just went to edit the file and it has utorrent_user_id: "999"
utorrent_group_id: "1001"
same as mine.

File: group_vars/all.yml (hunk 1 above)
Comment (bcurran3) [reply group_vars/all.yml:380]: Sorry for my total confusion. I see what you're talking about now. I have no idea now where I got that as it's the docker id and it's not what's referenced in the docker container setup guide: their default is 1000 for both.
I'll set it to 1000 now. If you think 0 is better, feel free to change it.

File: group_vars/all.yml (hunk 2 above)
Comment (bcurran3) [reply group_vars/all.yml:381]: Sorry for my total confusion. I see what you're talking about now. I have no idea now where I got that as it's the docker id and it's not what's referenced in the docker container setup guide: their default is 1000 for both.
I'll set it to 1000 now. If you think 0 is better, feel free to change it.

File: group_vars/all.yml (hunk 1 above)
Comment (bcurran3) [reply group_vars/all.yml]: I changed it to "0" to match the Transmission defaults to keep the two aligned

File: group_vars/all.yml (hunk 2 above)
Comment (bcurran3) [reply group_vars/all.yml]: I changed it to "0" to match the Transmission defaults to keep the two aligned

File: group_vars/all.yml (hunk 1 above)
Comment (davestephens) [reply group_vars/all.yml]: The UID you specify needs to match one on the host system, so what they say/recommend is totally irellevant. 0 is good though 👍

**Pull request merged**